cfg/1.13/master.yaml

```diff
@@ -556,19 +558,22 @@ groups:
     scored: true
 
   - id: 1.1.34
-    text: "Ensure that the --experimental-encryption-provider-config argument is
-    set as appropriate (Scored)"
+    text: "Ensure that the --encryption-provider-config argument is set as appropriate (Scored)"
     audit: "ps -ef | grep $apiserverbin | grep -v grep"
+    type: "manual"
     tests:
       test_items:
-      - flag: "--experimental-encryption-provider-config"
+      - flag: "--encryption-provider-config"
+        compare:
+          op: has
+          value: "</path/to/EncryptionConfig/File>"

```

Comment:
“</path/to/EncryptionConfig/File>” isn’t supposed to be a real location, it could be wherever the user has put the config file, and there is no default for this. That’s why we were just testing that the field is set, but not checking what value it’s set to. (I suppose we could be checking that it’s not set to an empty string.)

Reply:
OK Changed it :)